web-external/tests/compose.js

```diff
@@ -0,0 +1,79 @@
+
+import { createStore } from 'redux';
+import { compose } from '../src/utils/reducer';
+import { setScalar } from '../src/utils/common-reducers';
+
+import test from 'tape';
+
+
+test('compose duplicate leaves', (t) => {
+  t.plan(20);
```

Comment:
If you aren't doing any async tests, I find it's better to omit the call to `t.plan()` and instead put a `t.end()` after the final test (but this is a matter of taste - I like my method because it lets me change the number of tests without having to update the `t.plan()` call).